README.md

```diff
@@ -74,7 +79,29 @@ SPIDER=sc_florianopolis make run_spider
 
 ## Troubleshooting
 
-### "Permission denied" error when files are downloaded
+### Non-specific errors when building or running
+Several error is because your user don't is from docker group. To resolve this,
+execute the follow command and do logout and login:
+
+```console
+$ sudo usermod -aG docker $USER
+```
```

Comment:
Add a sentence to remember the users that they need to logout and login in the session

Reply:
The previous line is:

`execute the follow command and do logout and login`

It is not enough?